### ISSUE TITLE
No spaces after an opening parenthesis and before a closing parenthesis 

### ISSUE THREAD

**Issue by varelajp** [eslint/eslint#15100]
**What is the name of the syntax to implement?**
No spaces after an opening parenthesis and before a closing parenthesis 

**Please provide the TC39 URL for the syntax proposal:**

**Please provide some example code for the new syntax:**

```js
if ( obj.hasOwnProperty( el ) ) {
  objSum += obj[el];
}
```
must be corrected and forced automatically to
```js
if (obj.hasOwnProperty(el)) {
  objSum += obj[el];
}
```
just like you force ```obj[ el ]``` to ```obj[el]```

## Implementation Checklist

Please check off all items that have already been completed. Be sure to paste the pull request URLs next to each item so we can verify the work as done.

- [ ] Ecma262 update: 
- [ ] ESTree update: 
- [ ] Acorn update: 
- [ ] `eslint-visitor-keys` update: 
- [ ] `espree` update: 
- [ ] `eslint-scope` update: 
- [ ] `eslint` update: 

**Are you willing to submit a pull request to implement this syntax?**

**Comment by ljharb** [comment]
https://eslint.org/docs/rules/func-call-spacing should handle the function call, and https://eslint.org/docs/rules/space-in-parens should handle the `if`.

**Comment by varelajp** [comment]
Hi!

Using *npx eslint --fix script.js* on vscode I don't get the automatic fix
for trailing spaces on parenthesis in function calls and if.
I get the fix on square brackets.

Regards...
Juan Varela
+58 414 9994551

**Comment by ljharb** [comment]
Do you have those rules enabled and configured properly?

**Comment by varelajp** [comment]
Maybe not.

I use this procedure on my projects folders:
_npm init
npx eslint --init
npm link eslint-config-google
npx eslint --fix program.js_

Where can I configure rules options (e.g: **_"space-in-parens": ["error", "never"]_**)?

Thanks for your support...
Juan Varela

**Comment by ljharb** [comment]
You have to manually configure rules in your eslint config; the Google config won’t include everything.

**Comment by varelajp** [comment]
Ok.
I think I have to add rules in project's .eslintrc.json or, better yet, try to do it globally so as not to repeat.
After my commands project's _.eslintrc.json_ only have this content (no rules setted):
_{
    "env": {
        "browser": true,
        "es2021": true,
        "node": true
    },
    "extends": [
        "google"
    ],
    "parserOptions": {
        "ecmaVersion": 12
    },
    "rules": {
    }
}_

Maybe the rules are configured on my global _eslint-config-google_ installation.

Thanks again...
Juan Varela

**Comment by ljharb** [comment]
Definitely you don’t want anything global, including eslint itself; if a linter isn’t checked into the project, the project shouldn’t be linted.

**Comment by varelajp** [comment]
Yeah.
But, definitely you don't want a 3 MB folder for a script:
**_console.log(`Hello World!`);_**

That's my dilemma.

Thanks! :+1:

**Comment by ljharb** [comment]
Yes, you do, if you want it reliably linted :-)

**Comment by mdjermanovic** [comment]
As @ljharb answered, this rule already exists ([space-in-parens](https://eslint.org/docs/rules/space-in-parens)), so I'm closing this issue.
